arrow-libs/examples/src/test/kotlin/arrow/DataTypeExamples.kt

```diff
@@ -64,14 +65,14 @@ class DataTypeExamples : FreeSpec() {
         // Either http://arrow.io/docs/apidocs/arrow-core-data/arrow.core/-either/
         "Either left or right" - {
             fun parse(s: String): ProblemOrInt = try {
-                Either.right(s.toInt())
+                Right(s.toInt())
             } catch (e: Throwable) {
-                Either.left(invalidInt)
+                Either.Left(invalidInt)
```

Comment:
```suggestion
                Left(invalidInt)
```
**Nit:** Just to be consistent with the construction of the rest of `Left` values in the file